tests/src/test/kotlin/se/ansman/kotshi/TestAdapterGeneration.kt

```diff
@@ -156,4 +156,32 @@ class TestAdapterGeneration {
         assertEquals(GenericClassWithQualifier("Hello, world!"), actual)
         assertEquals(json, adapter.toJson(actual))
     }
-}
\ No newline at end of file
+
+    @Test
+    fun testMultipleJsonQualifiers() {
+        val adapter = Moshi.Builder().add(object : Any() {
```

Comment:
Could you wrap each statement on its own line.

Reply:
Yeah, happy to fix up the style (or when i next modify this file). Do you mean `add`, `build`, and `adapter` on new lines?

Reply:
Yeah, but I fixed it after I merged it so don't worry about it :)

Reply:
👍 thanks!